config/initializers/time.rb

```diff
@@ -1,7 +1,8 @@
 # Extends time class with convenience methods
 class Time
   def display_date
-    getlocal.strftime("%Y-%m-%d")
+    # not being converted to /
+    getlocal.strftime("%m/%d/%Y")
```

Comment:
Are you sure? Here's what I get when I try it in `rails console`:

```
irb(main):001:0> Time.now.display_date
=> "08/13/2017"
```

Looks right to me. Is there a trouble spot I can see it not working?